.gitignore

```diff
@@ -1,13 +1,25 @@
-**/*.aux
-**/*.bbl
-**/*.bcf
-**/*.blg
-**/*.dvi
-**/*.log
-**/*.out
-**/MOC.synctex.gz
-**/MOC.ps
-**/MOC.pdf
-**/MOC.fls
-**/MOC.fdb_latexmk
-role_diagram.svg
+
+#
+# Project specific excludes
+#
+#
+# ivoatex intermediate files
+*.aux
+*.bbl
+*.blg
+*.fdb_latexmk
+*.fls
+*.log
+*.out
+*.pdf
+*.toc
+*.lof
+*.lot
+ivoatexmeta.tex
+
+# OS
+.DS_Store
```

Comment:
Ah... I've not thought of Macs a lot so far.  If this .DS_Store is something you'd expect MacOS to create in such directories, could you make a PR to ivoatex (it's still called svn-ignore.tex there) to add it?

Sorting into topical areas is nice, too.

Reply:
Good idea.  I created ivoa-std/ivoatex#69. 

Thanks for the review!